orcid-web/src/main/resources/freemarker/includes/ng2_templates/oauth-authorization-ng2-template.ftl

```diff
@@ -164,6 +165,16 @@
                 </div><!--personal-account-login-->
                 <!-- END -->
             </div><!--login-->
+
```

Comment:
@leomendoza123 Add Togglz feature to allow enabling/disabling this code in the admin UI

Reply:
What should be shown if the toggle is off, nothing or just the regular login?